include/deal.II/multigrid/mg_coarse.h

```diff
@@ -28,6 +28,47 @@ DEAL_II_NAMESPACE_OPEN
 /*@{*/
 
 /**
+ * Coarse grid solver using preconditioners only. This is a little wrapper,
+ * transforming a preconditioner into
+ * a coarse grid solver.
+ *
+ * @author Denis Davydov, 2016.
+ */
+template<class VectorType = Vector<double> >
+class MGCoarseGridPreconditioner :  public MGCoarseGridBase<VectorType>
```

Comment:
Should we really call this method `Preconditioner`? I would find it more intuitive to call this method `MGCoarseGridFromSmoother` or similar to indicate that we take the `smooth` operation. When thinking about a preconditioner, my view would be that this class needs to be templated on an arbitrary `PreconditionerType` type that gets fed to the initialize class. Other opinions?